spec.md

```diff
@@ -378,7 +378,7 @@ It is therefore RECOMMENDED that implementations avoid such strings.
 | requires | array of strings | A list of permissions that the user would have to give the service, if they provision it. The only permissions currently supported are `syslog_drain`, `route_forwarding` and `volume_mount`. |
 | bindable* | boolean | Specifies whether Service Instances of the service can be bound to applications. This specifies the default for all plans of this service. Plans can override this field (see [Plan Object](#plan-object)). |
 | metadata | object | An opaque object of metadata for a Service Offering. It is expected that Platforms will treat this as a blob. Note that there are [conventions](profile.md#service-metadata) in existing Service Brokers and Platforms for fields that aid in the display of catalog data. |
-| dashboard_client | [DashboardClient](#dashboard-client-object) | Contains the data necessary to activate the Dashboard SSO feature for this service. |
+| dashboard_client | [DashboardClient](profile.md#dashboard-client-object) | Deprecated (see the [Cloud Foundry Service Metadata](profile.md#cloud-foundry-service-metadata) object). |
```

Comment:
I think this needs to point to the old stuff, until its fully removed, but keep the deprecation statement and pointer to the new stuff.

Reply:
See above. Since both `spec.md` and `profile.md` are versioned, can we not move things between these without 'breaking' anything?

Reply:
My point is that existing brokers might look at the spec and see this field and, even though its deprecated, they're still allowed to send it. Which means we need to still clearly define it.  We can't leave a hole in the supported functionality.  See how we dealt with org_id - we tagged it as deprecated in favor of context but still fully defined it.